### PR TITLE
Add console command shortcuts

### PR DESCRIPTION
**Pull request by i906**
This PR adds command shortcuts that are useful for those that only have a single type of driver.

Example Tenanti config:
```php
'drivers' => [
    'tenant-a' => [
        // ...
    ],
]
```

Shortcuts:
```
php artisan tenanti:make add_tenant_migration
php artisan tenanti:migrate
php artisan tenanti:tinker 1
```

Equivalent commands:
```
php artisan tenanti:make tenant-a add_tenant_migration
php artisan tenanti:migrate tenant-a
php artisan tenanti:tinker tenant-a 1
```

### CODE REVIEW

File: src/Console/BaseCommand.php [hunk 1]
```diff
@@ -49,14 +50,52 @@ protected function writeMigrationOutput(FactoryInterface $migrator)
     }
 
     /**
+     * Get driver argument or first driver in the config.
+     *
+     * @return string
+     */
+    protected function getDriver()
+    {
+        $argument = $this->argument('driver');
```
Comment (crynobone) [inline src/Console/BaseCommand.php]:
```php
$driver = $this->argument('driver') ?: $this->getDriverFromConfig();

if (! empty($driver)) {
    return $driver;
}
```

File: src/Console/MigrateMakeCommand.php [hunk 2]
```diff
@@ -59,11 +60,26 @@ public function __construct(TenantiManager $tenant, Creator $creator, Composer $
      */
     public function handle()
     {
+        $arg1 = $this->argument('driver');
+        $arg2 = $this->argument('name');
```
Comment (crynobone) [inline src/Console/MigrateMakeCommand.php]: I don't prefer `$arg1` and `$arg2`, might just use `$argument[0]` and `$argument[1]` (using array).

File: src/Console/MigrateMakeCommand.php [hunk 3]
```diff
@@ -59,11 +60,26 @@ public function __construct(TenantiManager $tenant, Creator $creator, Composer $
...
+            $name = $arg1;
+            $driver = $this->getDriverFromConfig();
+
+            if (empty($driver)) {
+                throw new RuntimeException('Not enough arguments (missing: "driver").');
+            }
+        } else {
+            $name = $arg2;
+            $driver = $arg1;
+        }
```
Comment (crynobone) [inline src/Console/MigrateMakeCommand.php]: This whole section should be on it's own method.

File: src/Console/TinkerCommand.php [hunk 4]
```diff
@@ -27,8 +28,23 @@ class TinkerCommand extends BaseCommand
...
+            $id = $arg1;
+            $driver = $this->getDriverFromConfig();
+
+            if (empty($driver)) {
+                throw new RuntimeException('Not enough arguments (missing: "driver").');
+            }
+        } else {
+            $id = $arg2;
+            $driver = $arg1;
+        }
```
Comment (crynobone) [inline src/Console/TinkerCommand.php]: Repeated code, should be a method.

**Comment by i906** [comment]
I have updated as per your review.

**Comment by crynobone** [comment]
Merged manually due to merge conflict.

**Pull request closed without merging**